test/e2e-tests.sh

```diff
@@ -106,7 +106,7 @@ function run_e2e_tests() {
   kubectl label namespace $2 istio-injection=enabled --overwrite
   local options=""
   (( EMIT_METRICS )) && options="-emitmetrics"
-  report_go_test -v -tags=e2e -count=1 ./test/$1 -dockerrepo gcr.io/knative-tests/test-images/$1 ${options}
+  report_go_test -v -tags=e2e -count=1 -timeout=20m ./test/$1 -dockerrepo gcr.io/knative-tests/test-images/$1 ${options}
```

Comment:
The 20 min timeout was added because the `TestAutoscaleUpDownUp_WithReadinessProbe` runs the same scenarios as the previous `TestAutoscaleUpDownUp` but with readinessProbe defined in the ConfigurationSpec. The `TestAutoscaleUpDownUp` takes a long time and doubling it upped it to about 17 mins and so was running into go test timeout failure. It exceeds the default 10 mins for `go test`

Maybe PR #1670 when merged will help reduce the time. But for now, the default timeout of 10 mins is no good. Happy to consider alternatives. Suggestions?

Reply:
Not an issue anymore. this commit from yesterday updated the timeout to 20 mins - https://github.com/knative/serving/pull/1702/commits/fbfa23dbf40e7231f9d076635b68a059d6ca3f69